### PR TITLE
Added femtofarad (fF) to the units.

### PR DESCRIPTION
**Pull request by Hannnsen**
Pull request to address issue  #209.
Four lines were added to define fF as a UnitQuantity.

### CODE REVIEW

File: quantities/units/electromagnetism.py [hunk 1]
```diff
@@ -137,6 +137,11 @@
     nF/1000,
     symbol='pF'
 )
+fF = UnitQuantity(
+    'femtofarad',
+    pf/1000,
```
Comment (apdavison) [inline quantities/units/electromagnetism.py:142]: this should be `pF/1000`

**Review by apdavison — CHANGES_REQUESTED** [review]
Many thanks for the pull request, just one small typo to fix

File: quantities/units/electromagnetism.py (hunk 1 above)
Comment (Hannnsen) [reply quantities/units/electromagnetism.py:142]: Thank you for spotting that! For some reason, I put nano instead of femto.

**Review by apdavison — APPROVED** [review]

File: quantities/units/electromagnetism.py (hunk 1 above)
Comment (Hannnsen) [reply quantities/units/electromagnetism.py]: There is a typo in your code. `pf/1000` will not work, no? So, it should be `pF/1000`.

**Pull request merged**